pkg/framework/guest.go

```diff
@@ -105,32 +106,63 @@ func (g *Guest) Setup() error {
 }
 
 func (g *Guest) WaitForAPIDown() error {
-	apiDown := func() error {
-		_, err := g.k8sClient.
-			CoreV1().
-			Services("default").
-			Get("kubernetes", metav1.GetOptions{})
-
-		if err == nil {
-			return microerror.Mask(fmt.Errorf("API up"))
+	time.Sleep(1 * time.Second)
+
+	log.Println("level", "debug", "message", "waiting for k8s API to be down")
```

Comment:
Note that I add logging here via a native lib. The output is n structured JSON but I want to introduce an injected micrologger in the next PRs. These should then be quite nice and easy.